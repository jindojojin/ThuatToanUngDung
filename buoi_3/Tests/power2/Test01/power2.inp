50
102 23 34 456 678 8 230 34 456 678 678 678 456 456 345 12 23 34 456 678 12 23 34 456 679 679 679 679 456 345 12 23 32 456 678 1024 23 34 456 680 680 680 680 680 345 1 2 16 5 5
