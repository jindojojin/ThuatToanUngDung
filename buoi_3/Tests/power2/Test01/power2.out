1 2 8 16 32 1024
